google/cloud/forseti/services/model/modeller.py

```diff
@@ -103,8 +103,20 @@ def delete_model(self, model_name):
 
         Args:
             model_name (str): name of the model to be deleted
+
+        Returns:
+            result (str): Success or failure message based on whether model is
+            deleted or not
         """
 
         LOGGER.info('Deleting model: %s', model_name)
         model_manager = self.config.model_manager
-        model_manager.delete(model_name)
+        try:
+            model_manager.delete(model_name)
+            result = 'SUCCESS'
```

Comment:
No need to have a return or the `finally`, just raise an exception and handle it from the caller method.